### DISCUSSION TITLE
Mac Support

### DISCUSSION THREAD

**Discussion by brunodeangelis** [IntentDev/PluginBuilder#3] [Q&A]
Hey, thanks for this!

I'm aware at the moment Mac is not supported. I haven't delved into the project yet so dunno what would be required to have it run on it, but I can help you out with testing if you want, I'm running an M1 Max on macOS 14.4.1 running the latest TD `2023.11600`

**Comment by keithlostracco** [comment]
Great yeah there will be few hoops to jump through. I just realized I might be able to get access to a relatively current Mac to test on. I won't be able to get to it for a few days but I'll let you know here when I have branch ready for you to test.
